example-schema/relationships/HasOlderSibling.yaml

```diff
@@ -0,0 +1,10 @@
+name: HasOlderSibling
```

Comment:
Would prefer this to be something like the following:

```
name: ReflexiveRichRelationship (https://www.merriam-webster.com/dictionary/reflexive)
relationship: HAS_RICHLY_RELATED_MAIN
properties:
  someProp:
    type: Word
    description: Word description.
    label: Word label
to:
  type: MainType
  hasMany: true
from:
  type: MainType
  hasMany: true
```

And also, for relationships to other types

```
name: RichRelationship 
relationship: HAS_RICHLY_RELATED_CHILD
properties:
  someProp:
    type: Word
    description: Word description.
    label: Word label
to:
  type: ChildType
  hasMany: true
from:
  type: MainType
  hasMany: true
```